static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -112,14 +114,18 @@ const show = (): Promise<boolean> => import(
 
         mountDynamic(
             container,
-            module.ExampleComponent,
+            module.DigitalSubscriberAppBanner,
             {
-                message: messageConfig.extras["test-key"],
-                onButtonClick: () => {
+                onButtonClick: (buttonId: number) => {
                     if (appboy) {
-                        appboy.logInAppMessageClick(messageConfig);
+                        const thisButton = new appboy.InAppMessageButton(null,null,null,null,null,null,buttonId)
```

Comment:
Would it be possible to know more about the shape of `InAppMessageButton`? `any` seems a little vague, and knowing whether it takes 5, 6 or 7 nulls seems a little obscure.